### DISCUSSION TITLE
Using electron-log with electron-vue in renderer process

### DISCUSSION THREAD

**Discussion by frifrafry** [megahertz/electron-log#318] [General]
I couldn't get electron-log working with electron-vue, as I was always running into:
`[Vue warn]: Property or method "toJSON" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.`

In the end I solved this via setting up a simple default implementation:
`Vue.prototype.toJSON = function () {
  return 'toJSON not defined'
}
`
Any thoughts on this? Is there a better way to solve this?

**Comment by megahertz** [comment]
Currently, that's not possible to disable that. When serializing an object recursively, the loggers calls toJSON of an object if it's defined. But I'll think about configuring it in v5.
